l10n_br_stock/hooks.py

```diff
@@ -81,9 +81,62 @@ def set_stock_warehouse_external_ids(env, company_external_id):
 
 def pre_init_hook(cr):
     """Import XML data to change core data"""
-
-    if not tools.config["without_demo"]:
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    if env.ref("base.module_base").demo:
         _logger.info(_("Loading l10n_br_stock warehouse external ids..."))
-        env = api.Environment(cr, SUPERUSER_ID, {})
         set_stock_warehouse_external_ids(env, "l10n_br_base.empresa_simples_nacional")
         set_stock_warehouse_external_ids(env, "l10n_br_base.empresa_lucro_presumido")
+
+
+def create_quants_for_inventory(cr, locations, products):
```

Comment:
A better name would be create_locations_quants

Reply:
done